data/json/items/ammo.json

```
@@ -298,8 +298,26 @@
     "color": "dark_gray",
     "description": "Plastic jacketed copper cable of the type used in small electronics.",
     "material": [ "plastic", "copper" ],
-    "volume": "750 ml",
-    "weight": "2 g",
+    "//": "Calculated off of 12 AWG, at 10cm length, using http://www.mtlexs.com/technical-specifications-details/40/weight-of-copper-wire.",
+    "volume": "437 ml",
+    "weight": "984 g",
```

```suggestion
    "volume": "110 ml",
    "weight": "5 g",
```
Or something larger taking the plastic jacket into account.

---

I was sure I changed this.